dist/.well-known/security.txt

```diff
@@ -0,0 +1,16 @@
+# If you would like to report or fix a bug
+# Create an issue or a pull request on the
+# Northstar Github page
+
+# Northstar Atlas (Master Server)
+https://github.com/R2Northstar/Atlas
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
```

Comment:
Looking at again, maybe we should also link the org as a whole? ^^

Reply:
So just https://github.com/R2Northstar?

Reply:
Updated